compiler/rustc_middle/src/dep_graph/dep_node.rs

```diff
@@ -104,14 +104,15 @@ macro_rules! define_dep_nodes {
     };
 }
 
+// We ignore the query ID here so pass a sentinel
 rustc_query_append!(define_dep_nodes![
     /// We use this for most things when incr. comp. is turned off.
-    [] fn Null() -> (),
+    [] fn Null 0() -> (),
```

Comment:
Maybe use 65535 instead? That doesn't match any existing query, so it is more likely to uncover bugs.

Reply:
That would probably make sense, yeah